src/exampleExplorer.ts

```diff
@@ -186,6 +187,12 @@ export class ExampleExplorer {
   async selectBoard(
       context: vscode.ExtensionContext, channel: vscode.OutputChannel,
       telemetryContext: TelemetryContext) {
+    if (RemoteExtension.isRemote(context)) {
+      const message =
+          `You are in a container now. You can open examples in a new window(click 'File -> New Window').`;

```

Comment:
The project is open in Docker container now, Please open a new window and rerun this command.

Reply:
Fixed.